students/biegon_piotr/lesson_03_functions/input_validation.py

```diff
@@ -0,0 +1,21 @@
+print("Input Validation\n")
+
+
+def collatz(number):
+    if number % 2 == 0:
+        return number // 2
+    else:
+        return 3 * number + 1
+
+
+while 1:
```

Comment:
Logic is wrong. This code asks user every time to get input, where the task is different:
> Then write a program that lets the user type in an integer and **that keeps calling collatz() on that number until the function returns the value 1.**

So for single input, a bunch of numbers should be printed, until `1` is reached.